tests/consensus_tests/test_force_delete_peer.py

```
@@ -0,0 +1,84 @@
+        f"{peer_api_uri}/cluster/peer/{peer_id}?force=true",
+    )
+    assert response.status_code == 200, f"Failed to force delete peer: {response.text}"
+
+
+def get_peer_id(peer_api_uri: str) -> int:
+    response = requests.get(f"{peer_api_uri}/cluster")
+    assert response.status_code == 200, f"Failed to get peer ID: {response.text}"
+    return response.json()["result"]["peer_id"]
+
+def test_force_delete_source_peer_during_transfers(tmp_path: pathlib.Path):
+    assert_project_root()
```

Ruff (0.8.2)

`assert_project_root` may be undefined, or defined from star imports

(F405)